wiki/installing-and-updating-drivers.md

```diff
@@ -0,0 +1,215 @@
+ <img src="./img/installing-and-updating-drivers/winre/startup_settings.png" alt="An image to click to startup options" height="350px">
+
+   After your computer restarts, press <kbd>4</kbd> to boot into Safe Mode, or <kbd>5</kbd> to boot into Safe Mode with internet access.
+
+On earlier versions of Windows:
+
+- Restart your computer and repeatedly press F8 before the Windows logo appears.
+- At the Advanced Boot Options menu, select “Safe Mode” to boot into Safe Mode, or “Safe Mode with Networking” to boot into Safe Mode with internet access.
+
+2. Uninstall the Driver in Safe Mode:
+
+Once in safe mode, you can uninstall the problematic driver:
+
+1. Open the Device Manager (search for it in the Start Menu).
+
+2. Expand the category for the device with the recently installed driver.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/device-manager-properties-button.png" alt="The context menu for a display adapter in Device Manager, with Properties highlighted." width="500px">
+
+3. In Device Manager, find the device that is causing problems (you might have to expand a category), right-click on it, and click "Properties".
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/device-manager-properties-roll-back-driver.png" alt="The properties window for a device in Device Manager, with the Driver tab and Roll Back Driver button highlighted." height="350px">
+
+4. Go to the "Driver" tab at the top of the window, and then click "Roll Back Driver".
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/driver-rollback-confirmation.png" alt="The confirmation dialogue presented after selecting Roll Back Driver." height="300px">
+
+5. Windows will ask you why you're rolling back to a previous driver. Select a reason, and click "Yes". If you want to, you can leave a detailed response in the "Tell Us More" field, at the bottom of the window.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/driver-rollback-confirmation.png" alt="The confirmation dialogue presented after selecting Roll Back Driver." width="500px">
+
+6. Windows will then restore your driver to the previous version, which could take up to 5-10 minutes.
+
+<img src="./img/installing-and-updating-drivers/troubleshooting/hardware-change-restart-prompt.png" alt="The prompt to restart your computer, shown after rolling back a driver." width="500px">
+
+
+Or if you can't access the normal safe mode, you may try and access Safe Mode with Command Prompt and execute the following commands in Command Prompt:
+
+1. ```Dism /image:C:/ /Get-Drivers```
+
+2. ```DISM /Image:C:/ /Remove-Driver /Driver:not_working_driver.inf``` (replace not_working_driver with the driver that isn't working).
+
+3. You should see something like this:  
+```
+::: tip Note
+If you have trouble with this, proceed to another troubleshooting method below.
+:::
+```
+::: Info:
+If something went wrong, proceed with another method of troubleshooting
+:::
```

Comment:
See what?

Also for the "if something went wrong" part you do not have to put it in a custom container (syntax is wrong anyway)